go/src/koding/kites/kloud/metrics/metrics.go

```diff
@@ -69,6 +73,12 @@ func (p *Publisher) Publish(r *kite.Request) (interface{}, error) {
 		return nil, err
 	}
 
+	go func() {
+		if _, err := http.Post(p.metricsEndpoint, "application/json", bytes.NewReader(r.Args.Raw)); err != nil {
```

Comment:
I have used default client since i wasnt handling error cases but as you said would be better to have a configured client in the case where the target is down/slow to respond